src/oc_bifrost/rebar.config

```diff
@@ -38,7 +38,7 @@
     {opscoderl_wm, ".*",
         {git, "git://github.com/chef/opscoderl_wm.git",               {branch, "master"}}},
     {sqerl, ".*",
-        {git, "git://github.com/chef/sqerl.git",                      {branch, "master"}}},
+        {git, "git://github.com/chef/sqerl.git",                      {branch, "lbaker/envy"}}},
```

Comment:
We should create a ticket to fork manderson/envy in chef and update chef-server to point to master of sqerl once that is done.

Reply:
We merged my sqerl PR to use chef/pooler vs seth pooler, so I pointed to sqerl master.